applications/adv_networking_bench/README.md

```diff
@@ -1,48 +1,54 @@
 # Advanced Networking Benchmark
 
-This is a simple application to measure a lower bound on performance for the advanced networking operator
-by receiving packets, optionally doing work on them, and freeing the buffers. While only freeing the packets is
-an unrealistic workload, it's useful to see at a high level whether the application is able to keep up with
-the bare minimum amount of work to do. The application contains both a transmitter and receiver that are
-designed to run on different systems, and may be configured independently.
+> [!TIP]
+> Review the [High Performance Networking tutorial](/tutorials/high_performance_networking/README.md) for guided
+> instructions to configure your system and test the Advanced Network Library.
+
+This is a sample application to measure a lower bound on performance for the [Advanced Network
```

Comment:
Library is capitalized above but not here

Reply:
keeping all lowercase, ty